app/scripts/metamask-controller.js

```diff
@@ -4753,8 +4769,8 @@ export default class MetamaskController extends EventEmitter {
 
       try {
         const securityProviderResponse = await securityProviderCheck(
-          requestData,
-          methodName,
+          _requestData,
+          _methodName,
```

Comment:
Now that code fencing is wrapping the entire method, this args do not need to be prefixed with `_`